docs/content/configuration/csi_ug_config_create_storageclasses_topology.md

```diff
@@ -0,0 +1,49 @@
+# Creating a StorageClass with topology feature
+
+Create a separate storage class yaml file must be created for StorageClass to be Topology Aware.
```

Comment:
I think the grammar is not right here, maybe drop the `Create`?
but even so, a separate storage class file must be created for storageclass to be topology aware? did you mean a separate secret file?

Reply:
DONE - @ArbelNathan  please approve so @oriyarde  can resolve

Reply:
LGTM